SwanProjects/src/Components.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) SWAN Development Team.
+// Author: Omar.Zapata@cern.ch 2021
+import { classes, LabIcon } from '@jupyterlab/ui-components';
+import * as React from 'react';
+import ReactTooltip from 'react-tooltip';
+
```

Comment:
What does this file contain? Was this copied from somewhere or written by you?

Reply:
I did a clarification in the documentation of the file, 
I did the HelpTooltip component using the external dependency "react-tooltip"  to display information about the release and platform in the project dialog.
For the Card, it is based in the code from launcher's Card in JupyetrLab which is not exported,
with some small modification for ours needs.